### PR TITLE
Release v1.6.2

### PR DESCRIPTION
**Pull request by asmacdo**
v1.6.2 release commit

### CODE REVIEW

File: website/content/en/docs/upgrading-sdk-version/v1.6.2.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: v1.6.2
+weight: 998993998
+---
+
+## Add the manager config patch to config/default/kustomization.yaml
+
+The scaffolded `--config` flag was not added to either ansible-/helm-operator binary when [config file](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html) support was originally added, so does not currently work. The `--config` flag supports configuration of both binaries by file; this method of configuration only applies to the underlying [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager), not the operator as a whole. To optionally configure the operator's Deployment with a config file, make the following update to config/default/kustomization.yaml:
+```diff # If you want your controller-manager to expose the /metrics # endpoint w/o any authn/z, please comment the following line. - manager_auth_proxy_patch.yaml + +# Mount the controller config file for loading manager configurations +# through a ComponentConfig type +- manager_config_patch.yaml ```
```
Comment (varshaprasad96) [inline website/content/en/docs/upgrading-sdk-version/v1.6.2.md:9]: I think formatting issue with the generator
```suggestion
```diff 
# If you want your controller-manager to expose the /metrics 
# endpoint w/o any authn/z, please comment the following line. 
- manager_auth_proxy_patch.yaml 
+ +# Mount the controller config file for loading manager configurations 
+# through a ComponentConfig type 
+- manager_config_patch.yaml ```
```

File: website/content/en/docs/upgrading-sdk-version/v1.6.2.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: v1.6.2
+weight: 998993998
+---
+
+## Add the manager config patch to config/default/kustomization.yaml
+
+The scaffolded `--config` flag was not added to either ansible-/helm-operator binary when [config file](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html) support was originally added, so does not currently work. The `--config` flag supports configuration of both binaries by file; this method of configuration only applies to the underlying [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager), not the operator as a whole. To optionally configure the operator's Deployment with a config file, make the following update to config/default/kustomization.yaml:
```
Comment (varshaprasad96) [inline website/content/en/docs/upgrading-sdk-version/v1.6.2.md:8]:
```suggestion
The scaffolded `--config` flag was not added to either ansible-/helm-operator binary when [config file](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html) support was originally added, so it does not currently work. The `--config` flag supports configuration of both binaries by file; this method of configuration only applies to the underlying [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager), not the operator as a whole. To optionally configure the operator's Deployment with a config file, make the following update to `config/default/kustomization.yaml`:
```

**Review by varshaprasad96 — APPROVED** [review]
just a nit and lgtm

**Review by jmrodri — APPROVED** [review]
/lgtm

File: website/content/en/docs/upgrading-sdk-version/v1.6.2.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+## Add the manager config patch to config/default/kustomization.yaml
+
+The scaffolded `--config` flag was not added to either ansible-/helm-operator binary when [config file](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html) support was originally added, so it does not currently work. The `--config` flag supports configuration of both binaries by file, this method of configuration only applies to the underlying [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager), not the operator as a whole. To optionally configure the operator's Deployment with a config file, make the following update to `config/default/kustomization.yaml`:
+```diff 
+# If you want your controller-manager to expose the /metrics 
+# endpoint w/o any authn/z, please comment the following line. 
+- manager_auth_proxy_patch.yaml 
++ +# Mount the controller config file for loading manager configurations 
++# through a ComponentConfig type 
++- manager_config_patch.yaml ```
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.6.2.md:15]:
````suggestion
+- manager_config_patch.yaml
```
````

File: website/content/en/docs/upgrading-sdk-version/v1.6.2.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+weight: 998993998
+---
+
+## Add the manager config patch to config/default/kustomization.yaml
+
+The scaffolded `--config` flag was not added to either ansible-/helm-operator binary when [config file](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html) support was originally added, so it does not currently work. The `--config` flag supports configuration of both binaries by file, this method of configuration only applies to the underlying [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager), not the operator as a whole. To optionally configure the operator's Deployment with a config file, make the following update to `config/default/kustomization.yaml`:
+```diff 
+# If you want your controller-manager to expose the /metrics 
+# endpoint w/o any authn/z, please comment the following line. 
+- manager_auth_proxy_patch.yaml
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.6.2.md:12]: This is just a list element, silly diff
```suggestion
\- manager_auth_proxy_patch.yaml 
```

**Review by estroz — APPROVED** [review]
One suggestion about noting security fixes, then LGTM

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**